### PR TITLE
OCPBUGS2214: Deploying an extension rpm with a MachineConfig and a custom osImage at the same time breaks the machine config pool

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OCPBUGS-2214

Preview: https://57245--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/coreos-layering.html

Update the wording in the IMPORTANT note with the [Known Issue in the Custom Content on RHCOS TE slides](https://docs.google.com/presentation/d/1g2Y4lTZG9w5--X5G-emAzu7p-a5oFkKQXsDAYV-VPX8/edit#slide=id.g1165ecc17b8_0_0) 

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sinnykumari** [comment]
Thanks @mburke5678 for quick fix
/lgtm

**Comment by mburke5678** [comment]
@sergiordlr PTAL. .

**Comment by mburke5678** [comment]
@sergiordlr Can you PTAL at this small change? Thank you in advance.

**Comment by mburke5678** [comment]
@sergiordlr Can you PTAL at this small change? Thank you in advance.

**Comment by sergiordlr** [comment]
@mburke5678 It looks good to me.

Thank you!!

File: post_installation_configuration/coreos-layering.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ With {op-system} image layering, you can install RPMs into your base image, and
 
 [IMPORTANT]
 ====
-RPMs installed through a custom layered image can conflict with RPMs installed by using a machine config. It is recommended to use either machine config or a custom layered image to add extensions, but not both, unless you are certain there will be no conflicts. If there is a conflict, the MCO enters a `degraded` state when it tries to install the machine config RPM. You need to remove the conflicting extension from your machine config before proceeding.
+It is recommended to not apply realtime kernel and extensions RPMs as custom layered content. RPMs installed through a custom layered image can conflict with RPMs installed by using a machine config. If there is a conflict, the MCO enters a `degraded` state when it tries to install the machine config RPM. You need to remove the conflicting extension from your machine config before proceeding.
```
Comment (ousleyp) [inline post_installation_configuration/coreos-layering.adoc:20]: > It is recommended to not apply

I think that this phrasing is hard to follow, and some of the existing text in this IMPORTANT block could be clarified as well. At minimum, please change `It is recommended to not apply` to `It is not recommended to apply` or similar. 

Here's a suggested rewrite for you to consider:

Suggestion:
```suggestion
Installing realtime kernel and extensions RPMs as custom layered content is not recommended. This is because these RPMs can conflict with RPMs installed by using a machine config.

If there is a conflict, the MCO enters a `degraded` state when it tries to install the machine config RPM. To resolve this, remove the conflicting extension from the machine config before proceeding.
```

**Review by ousleyp — COMMENTED** [review]
This mostly LGTM, but I have a nit with the phrasing and a couple suggestions for clarity. Thanks!
